api/src/db/DynamoBusinessesDataClient.test.ts

```diff
@@ -70,23 +67,15 @@ describe("DynamoBusinessesDataClient", () => {
 
     const expectedValue = await dynamoBusinessesDataClient.findByBusinessName(businessName);
     expect(expectedValue).toBeDefined();
-    const { version, ...filteredExpectedValue } = expectedValue as unknown as {
-      version?: string;
-      [key: string]: any;
-    };
-    expect(filteredExpectedValue).toEqual(businessData);
+    expect(expectedValue).toEqual(businessData);
   });
 
   it("finds all businesses by the naicsCode", async () => {
     expect(await dynamoBusinessesDataClient.findAllByNAICSCode("some-naicsCode")).toHaveLength(0);
     await dynamoBusinessesDataClient.put(businessData);
     const expectedValue = await dynamoBusinessesDataClient.findAllByNAICSCode(naicsCode);
     expect(expectedValue[0]).toBeDefined();
-    const { version, ...filteredExpectedValue } = expectedValue[0] as unknown as {
-      version?: string;
-      [key: string]: any;
-    };
-    expect(filteredExpectedValue).toEqual(businessData);
+    expect(expectedValue[0]).toEqual(businessData);
```

Comment:
sand - does it make sense to have an assertion where expectedValue has a length of 1?

Reply:
The .toHaveLength(1) assertion is not applicable here because findByEncryptedTaxId returns an object, not an array or iterable. Objects do not have a length property, so this assertion will fail. The correct approach is to use .toEqual(businessData) to verify that the returned object matches the expected data